ntoskrnl/ex/sysinfo.c

```diff
@@ -11,12 +11,18 @@
 /* INCLUDES *****************************************************************/
 
 #include <ntoskrnl.h>
+#include <wmidata.h>
+#include <wmistr.h>
 #define NDEBUG
 #include <debug.h>
 
 /* The maximum size of an environment value (in bytes) */
 #define MAX_ENVVAL_SIZE 1024
 
+#define ACPI 0x41435049
+#define FIRM 0x4649524D
+#define RSMB 0x52534D42
```

Comment:
Using something like `'RSMB'` may help human-readability ^^

Reply:
I've already tried that, in kernel mode it would be represented backwards... :D

I.e. `IPCA`, `MRIF`, and `BMSR`... :sweat_smile:

Reply:
Correct.